db/repair.cc

```diff
@@ -215,12 +215,21 @@ class Repairer {
       // created before we process WAL files
       ExtractMetaData();
 
+      // Recover epoch_number of existing SST files and next epoch_number
+      if (HasMissingEpochNumber()) {
```

Comment:
[Note]
Unfortunately, I couldn't get RepairDB() fails on ingested file cuz those files are seen as corrupted due to https://github.com/facebook/rocksdb/blob/7.7.fb/db/repair.cc#L546-L555 and hence are skipped.

ImportColumnFamily also didn't work with RepairDB cuz https://github.com/facebook/rocksdb/blob/7.7.fb/db/repair.cc#L524 will returned the exporting CF id instead of the created CF id. Then we will have same files of same epoch number under the same CF, violating my added check https://github.com/facebook/rocksdb/pull/10922/files#diff-a6aa385940793f95a2c5b39cc670bd440c4547fa54fd44622f756382d5e47e43R422